zerver/tests/test_upload.py

```diff
@@ -255,9 +255,14 @@ def test_serve_file_unauthed(self) -> None:
         self.logout()
         response = self.client_get(url)
         self.assertEqual(response.status_code, 200)
-        remove_ratelimit_rule(86400, 1000, domain="spectator_attachment_access_by_file")
+
+        # Check that the /download/ variant works as well
+        download_url = url.replace("/user_uploads/", "/user_uploads/download/")
+        response = self.client_get(download_url)
+        self.assertEqual(response.status_code, 200)
```

Comment:
Do we want to test rate limiting for the `/download/` variant too?

Reply:
The variant I pushed a fe days ago adds these tests.